webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -11,6 +11,8 @@ import { translate as __ } from 'foremanReact/common/I18n';
 import TooltipButton from '../../../components/TooltipButton';
 import { LoadingState } from '../../../components/LoadingState';
 import { Table } from '../../../components/pf3Table';
+import { getDocsURL } from 'foremanReact/common/helpers';
```

Comment:
I think, you can combine this one with line 7.

Reply:
Wow, how did I miss that?

Reply:
Luckily, the CI notices such things ;)